.github/workflows/pr-root-sync.yaml

```diff
@@ -29,6 +29,28 @@ jobs:
       - name: Install nomos
         run: gcloud components install nomos
 
-      - name: Check manifests
+      - name: Check Config Connector resources
         run: nomos vet --clusters "" --no-api-server-check --source-format unstructured
         # Refer to the docs https://cloud.google.com/anthos-config-management/docs/how-to/nomos-command#vet
+
+      - name: Check manifests in base/
+        working-directory: ./root-sync/base
+        run: |
+          set -euxo pipefail
+          kustomize build . \
+          | kubeconform \
+            -verbose -strict \
+            -ignore-missing-schemas \
+            -schema-location default \
+            -schema-location 'https://raw.githubusercontent.com/datreeio/CRDs-catalog/main/{{.Group}}/{{.ResourceKind}}_{{.ResourceAPIVersion}}.json'
```

Comment:
Just curious; given that bases are always overlayed with `./root-sync/overlays/test`, is there any specific reason we want to validate them separately other than a `kustomize build` on the overlays?

Reply:
Good question! My rationale was:

- We can expect both to be valid, although be **base/** may be incomplete 🤔 
- We want to report problems separately to speed to troubleshooting

Reply:
Regarding incomplete `base/`; I think we might already have an example for it. 

Check out the following spec in https://github.com/PHACDataHub/sci-portal/pull/95 -
https://github.com/PHACDataHub/sci-portal/blob/0ac7d4394325dbfa79cfadf7a6db37666736e469/root-sync/base/istio-ingress/certificate.yaml#L1-L15

The above would probably fail a schema check cause it's missing a `dnsNames` key, which gets patched as an overlay - 
https://github.com/PHACDataHub/sci-portal/blob/0ac7d4394325dbfa79cfadf7a6db37666736e469/root-sync/overlays/test/istio-ingress/patch-certificate.yaml#L5-L7